### PR TITLE
MYFACES-4435 - Update Language 2.3-next

### PR DESCRIPTION
**Pull request by volosied**
[MYFACES-4435](https://issues.apache.org/jira/browse/MYFACES-4435)

### CODE REVIEW

File: impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java [hunk 1]
```diff
@@ -198,7 +198,7 @@ public String getUserAgent(FacesContext facesContext)
     /**
      * Users can overload this method to define in which scenarios a request should result
      * in an 'intercepted' page with proper windowId detection. This can e.g. contain
-     * blacklisting some userAgents.
+     * blocklisting some userAgents.
```
Comment (melloware) [inline impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java:201]: isn't blacklisting the correct word here?

**Review by melloware — CHANGES_REQUESTED** [review]

File: impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java (hunk 1 above)
Comment (volosied) [reply impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java:201]: Both mean the same thing, but the connotations are different.  It relates to blacklisting/whitelisting, which can be viewed via a racial lens.

Comment (melloware) [reply impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java:201]: Got it!

**Review by melloware — APPROVED** [review]

File: impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java (hunk 1 above)
Comment (mkienenb) [reply impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java:201]: "blocklisting" is not a word.   If you want to replace "blacklist", then pick a real word or phrase to do so.   Not everyone speaks English as a first language, and if you start picking words that don't appear in dictionaries, you will be doing far more excluding than including.

Comment (melloware) [reply impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java:201]: Yes I would suggest rewording it rather than using a non-existent word.

**Comment by melloware** [comment]
Just to close the loop on this I found this article:  https://www.vice.com/en/article/v7dd3d/we-need-to-stop-saying-blacklist-and-whitelist

> The developers of Chromium, the open source codebase that underpins Google’s Chrome browser, as well as the competing Microsoft’s Edge, [announced in 2019](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/inclusive_code.md) that “terms such as ‘blacklist’ and ‘whitelist’ reinforce the notion that black==bad and white==good.” That’s why since then its developers are using “blocklist” and “allowlist.”

Basically good enough for Google the word `blocklist` will become the new word 😄
